WoT-stats/src/app/stats/stats.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component } from '@angular/core';
+import { ApiService } from '../service/api.service';
+import { PlayerStats } from '../interfaces/playerStats';
+import { PlayerId } from '../interfaces/playerId';
+import { FormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-stats',
+  standalone: true,
+  imports: [
+    FormsModule,
+    CommonModule
+  ],
+  templateUrl: './stats.component.html',
+  styleUrl: './stats.component.scss'
+})
+export class StatsComponent {
+  playerName: string = '';
+  playerStats: PlayerStats | null = null;
+  playerId: PlayerId | undefined;
+  globalRating: number = 0;
+  battles: number = 0;
+  wins: number = 0;
+  roundedWinPercentage: string = '';
+
+  constructor(private apiService: ApiService) {}
+
+  searchPlayer() {
+    this.apiService.getPlayerId(this.playerName).subscribe(
+      (response: PlayerId) => {
+        if(response.status === 'ok' && response.data.length > 0) {
```

Comment:
If you have some if block the better way to do ` if(false) {return 'No data'}
your code when true.
`